apps/meteor/app/slashcommands-create/server/server.ts

```diff
@@ -1,61 +1,62 @@
 import { Meteor } from 'meteor/meteor';
 import { TAPi18n } from 'meteor/rocketchat:tap-i18n';
-import { IMessage } from '@rocket.chat/core-typings';
 
 import { settings } from '../../settings/server';
 import { Rooms } from '../../models/server';
 import { slashCommands } from '../../utils/lib/slashCommand';
 import { api } from '../../../server/sdk/api';
 
-function Create(_command: 'create', params: string, item: IMessage): void {
-	function getParams(str: string): string[] {
-		const regex = /(--(\w+))+/g;
-		const result = [];
-		let m;
-		while ((m = regex.exec(str)) !== null) {
-			if (m.index === regex.lastIndex) {
-				regex.lastIndex++;
+slashCommands.add(
+	'create',
+	function Create(_command: 'create', params, item): void {
```

Comment:
Nesting this function declaration as a parameter expression damages readability, as evidenced by the indentation increase.